examples/neuralNetwork-sequence-hand-gesture-load-model/index.html

```diff
@@ -0,0 +1,28 @@
+<!--
```

Comment:
@shiffman @MOQN *Very unimportant*, but I thought I raise this here before I forget: was wondering if we prefer `neuralNetwork-sequence-...` or a (shorter) `neuralSequence-...`? (Feeling the former is truer to the hierarchy - but maybe too long?)